src/test/java/io/stargate/sgv2/jsonapi/exception/JsonApiExceptionTest.java

```diff
@@ -30,7 +30,6 @@ public void happyPath() {
               error -> {
                 assertThat(error.message()).isEqualTo("Provided command unknown");
                 assertThat(error.fields())
-                    .hasSize(2)
```

Comment:
No need to check exact count: ErrorsV2 will add more entries. Same for other `hasSize()` cases.